src/components/Popup/Popup.tsx

```diff
@@ -20,17 +20,21 @@ const PopupButtonArea = ({
   cancelText,
   onCancel,
   onConfirm,
+  isWarning = false,
 }: {
   confirmText: string;
   cancelText: string;
   onCancel: () => void;
   onConfirm: () => void;
+  isWarning?: boolean;
```

Comment:
이 부분 isWarning 대신 type 받는 거 어떤가용??
실제 브라우저에서도 `alert`, `prompt`, `confirm` 이 세 개 팝업을 제공하고 있어서 동일하게 하면 편할 것 같아요!

```suggestion
  type = "confirm",
}: {
  confirmText: string;
  cancelText: string;
  onCancel: () => void;
  onConfirm: () => void;
  type?: 'confirm' | 'alert';
```